src/items/enumerations.md

```diff
@@ -122,6 +165,53 @@ enum OverflowingDiscriminantError2 {
 }
 ```
 
+### Accessing Discriminant Values
+
+#### Casting
+
+If there is no data attached to *any* of the variants of an enumeration, then
```

Comment:
There's no need to describe a field-less enum here. Just use the term.

```suggestion
If the enum is a field-less enum, then
```

Reply:
Per https://github.com/rust-lang-nursery/reference/pull/639#issuecomment-511515290, these terms may not be completely interchangeable. If "field-less" is meant to be synonymous to "C-like", then this change would be incorrect.